cms/envs/common.py

```diff
@@ -703,3 +709,75 @@
         'boilerplate_name': None,
     }
 ]
+
```

Comment:
Is the expectation that edx_notifications will always be installed/available (but not necessarily enabled) in the platform?  Just wondering if it should be included in OPTIONAL_APPS versus INSTALLED_APPS.  Also, this seems like a lot of configuration information to add into common.py -- it would be nice we had some sort of convention for  compartmentalizing it better, such as a separate 'notifications_config.py' module that was imported.